client/README.md

```diff
@@ -1,36 +1,64 @@
 # BigQuery Migration Service Batch SQL Translator CLI
 
-The [BigQuery Migration Service][BQMS] includes a product known as the
-[batch SQL translator][batch SQL translator] that supports translation of a
-wide variety of SQL dialects to Google Standard SQL. This command line tool
-simplifies the process of using the batch SQL translator by providing a
-framework for pre and postprocessing of SQL files as well as upload/download of
-SQL files to/from GCS.
+This directory contains the Batch SQL Translator CLI, a command line client
+that simplifies the process of using the [batch SQL translator][batch SQL translator]
+of the [BigQuery Migration Service][BQMS] to translate from a wide variety of
+SQL dialects to Google Standard SQL. This client provides a framework that
+handles pre and postprocessing of SQL files, upload/download of SQL files
+to/from GCS, and invoking the GCP SQL translation API.
+
+Read in the Google Cloud documentation [how to submit a translation job using
+this client][how to submit].
 
-- [Quickstart](#quickstart)
 - [Installation](#installation)
-- [Basic Usage](#basic-usage)
+- [Quickstart](#quickstart)
+- [Running Using `run.sh`](#running-using-runsh)
+- [Running Using `bqms-run` Directly](#running-using-bqms-run-directly)
   - [Environment Variables](#environment-variables)
-  - [config.yaml](#configyaml)
-  - [Recommended Usage](#recommended-usage)
+- [config.yaml](#configyaml)
 - [Metadata Lookup](#metadata-lookup)
   - [DDL and Metadata Dump](#ddl-and-metadata-dump)
   - [Unqualified References](#unqualified-references)
 - [Pre and Postprocessing](#pre-and-postprocessing)
   - [Handling Macro/Templating Languages](#handling-macrotemplating-languages)
+  - [Extraction of Heredoc SQL Statements from KSH Inputs](#extraction-of-heredoc-sql-statements-from-ksh-inputs)
   - [Custom Pre and Postprocessing Logic](#custom-pre-and-postprocessing-logic)
 - [Renaming SQL Identifiers](#renaming-sql-identifiers)
 - [Deploying to Cloud Run](#deploying-to-cloud-run)
 - [Performance](#performance)
 
-## Quickstart
+## Installation
+
+Preferred OS: Linux or macOS. Windows usage may be possible through PowerShell,
+but it is not officially supported.
+
+**Download and extract the latest release zip `dwh-migration-tools-vX.X.X.zip`
+from [the Releases page](https://github.com/google/dwh-migration-tools/releases).**
+
+Python &ge; 3.7.2 is required, as well as the additional local dependencies
+`pkg-config` and `libicu-dev`. Typical commands for installing these
+dependencies on Linux would be:
```

Comment:
Nit: "Debian-based Linux" would be more appropriate here given the use of `apt` below, but anyone who really needs that clarification is probably going to be lost anyway, so up to you.

Reply:
(it seems like at least one customer is interested in RHEL, so this may need to be split out into separate sections eventually)

Reply:
@mjquinn-google +1, it alway introduce debate on `what should be the preference linux distro`. I suggest we only provide the dependency package name and let reader to find their own way to get it install.